File: LeetCode/Easy/1480 RunningSumOf1dArray/1480RunningSumOf1dArray.py
from typing import List

class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        for i in range(1, len(nums)):
            nums[i] += nums[i - 1]
        return nums

if __name__ == "__main__":
    sol = Solution()
    nums = [1, 2, 3, 4]
    resultado = sol.runningSum(nums)
    print(resultado)

File: LeetCode/Easy/1480 RunningSumOf1dArray/1480RunningSumOf1dArray.cpp
#include <iostream>
#include <vector>

using namespace std;

class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        for (size_t i = 1; i < nums.size(); ++i) {
            nums[i] += nums[i - 1];
        }
        return nums;
    }
};

int main() {
    Solution sol;
    vector<int> nums = {1, 2, 3, 4};
    vector<int> resultado = sol.runningSum(nums);

    cout << "[";
    for (size_t i = 0; i < resultado.size(); ++i) {
        cout << resultado[i] << (i == resultado.size() - 1 ? "" : ", ");
    }
    cout << "]" << endl;

    return 0;
}